### DISCUSSION TITLE
3.1.2 Interaction of Roles - UML diagram

### DISCUSSION THREAD

**Discussion by xdxxxdx** [International-Data-Spaces-Association/IDS-RAM_4_0#220] [Ideas]
Hello IDS team.

In the page:
https://docs.internationaldataspaces.org/ids-ram-4/layers-of-the-reference-architecture-model/3-layers-of-the-reference-architecture-model/3-1-business-layer/3_1_2_interaction_of_roles

The UML diagram shows that it is the **Data provider search metadata, and Data consumer publish metadata.**
From my idea, it should be the other direction:  **Data provider publish metadata, and Data consumer search metadata.**
please see: ![Diagram](https://user-images.githubusercontent.com/15314807/208390057-ab394743-7d38-4d4b-afbf-567449da4290.png) [Please correct me if I made a mistake]

In the RMA3.0, the diagram was showing  **Data provider publish metadata, and Data consumer search metadata.**
please refer to the paper:  **Data provider publish metadata, and Data consumer search metadata.**
[IDSA-Position-Paper-GAIA-X-and-IDS.pdf](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/files/10257837/IDSA-Position-Paper-GAIA-X-and-IDS.pdf)

Thanks
XD

**Comment by ssteinbuss** [comment]
Hey @xdxxxdx ,

thanks for the hint and you are totally right. We will update the figure.

**Reply by ssteinbuss** [reply]
This one is fixed with the [latest release](https://docs.internationaldataspaces.org/ids-ram-4/layers-of-the-reference-architecture-model/3-layers-of-the-reference-architecture-model/3-1-business-layer/3_1_2_interaction_of_roles)

**Reply by xdxxxdx** [reply]
Thanks for the update !
